src/main/java/network/brightspots/rcv/HartCvrReader.java

```diff
@@ -109,10 +118,31 @@ private void readCastVoteRecord(List<CastVoteRecord> castVoteRecords, Path path)
         ArrayList<Pair<Integer, String>> rankings = new ArrayList<>();
         if (contest.Options != null) {
           for (Option option : contest.Options) {
-            String candidateName = option.Id;
-            if (candidateName.equals(source.getUndeclaredWriteInLabel())) {
-              candidateName = Tabulator.UNDECLARED_WRITE_IN_OUTPUT_LABEL;
+
+            //Can be null with some write-ins
+            String candidateNameClean = Objects.requireNonNullElse(option.Name, "");
```

Comment:
How far can we get without setting this to empty string? If it's null, I think it makes more sense to keep it null as long as possible -- unless that causes a ton of issues everywhere. Another idea is to set the null candidate name to the ID instead.